packages/stack/src/views/Stack/CardContainer.tsx

```diff
@@ -128,6 +130,38 @@ function CardContainer({
 
   const { colors } = useTheme();
 
+  const [pointerEvents, setPointerEvents] = React.useState<'box-none' | 'none'>(
+    'box-none'
+  );
+
+  const valueListenerCallback = React.useCallback(
+    ({ value }: { value: number }) => {
+      const newIsUnderEpsilon = value <= EPSILON ? 'box-none' : 'none';
+      if (newIsUnderEpsilon !== pointerEvents) {
+        setPointerEvents(newIsUnderEpsilon);
+      }
+    },
+    [pointerEvents]
+  );
+
+  const listenerRef = React.useRef<string>();
+
+  React.useEffect(() => {
+    if (scene.progress.next) {
+      // @ts-ignore
+      listenerRef.current = scene.progress.next.addListener(
+        valueListenerCallback
+      );
+    }
+    return () => {
+      if (scene.progress.next) {
+        // @ts-ignore
+        scene.progress.next.removeListener(listenerRef.current);
```

Comment:
Use optional chaning

```suggestion
        scene.progress.next?.removeListener(listenerRef.current);
```